packages/core/src/components/TextField/Styled/TextField.styled.tsx

```diff
@@ -77,9 +87,9 @@ export const Input = styled('input')<StyledProps>`
     &:not(:placeholder-shown) ~ ${Label}, &:focus ~ ${Label} {
         opacity: ${({ size }) => size === 'S' && '0'};
         ${({ size }) => size === 'M' && transformLabel};
-    }
```

Comment:
I guess by mistake you have removed this

Reply:
Good catch - fixed here: 449d986